.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parser": "babel-eslint",
+  "extends": "standard",
+  "globals": {
+    "atom": false,
+    "afterEach": false,
+    "beforeEach": false,
+    "describe": false,
+    "fdescribe": false,
+    "xdescribe": false,
+    "expect": false,
+    "it": false,
+    "fit": false,
+    "xit": false,
```

Comment:
Those are the same globals that Atom declares, but sure. https://github.com/atom/atom/blob/e3790b8f20f96e17c2ad5cad3ed53b41e4f23377/package.json#L162-L177